tests/xtimer_now_irq/README.md

```diff
@@ -0,0 +1,2 @@
+This test will print an error when the time is incorrect while interrups are disabled.
```

Comment:
Please change to something more specific like: This test will print an error when xtimer_now returns an invalid value when called while interrupts are disabled.